remotable.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.add_dependency "activeresource"
   s.add_dependency "activerecord"
   s.add_dependency "activesupport"
-  
-  s.add_development_dependency "rails", ">= 3.1.0", "< 5.0.0"
+
+  s.add_development_dependency "rails", ">= 3.1.0", "< 5.1.0"
```

Comment:
Any reason you went with `< 5.1.0` versus `< 6.0.0`? Does Remotable not work on `5.1.0`?